site/docs/uk/guide/reactions.md

```diff
@@ -92,9 +116,13 @@ grammY надає вам можливість ще більше відфільт
 
 ```ts
 // Оновлення, в яких поточна реакція містить принаймні один емодзі.
-bot.on("message_reaction:new_reaction:emoji", (ctx) => {/* ... */});
+bot.on("message_reaction:new_reaction:emoji", (ctx) => {
+  /* ... */
+});
 // Оновлення, в яких попередня реакція містила принаймні один користувацький емодзі.
-bot.on("message_reaction:old_reaction:custom_emoji", (ctx) => {/* ... */});
+bot.on("message_reaction:old_reaction:custom_emoji", (ctx) => {
+  /* ... */
+});
```

Comment:
Please revert these changes

Reply:
I can't, because it's result of `npm run docs:fmt`.